### PR TITLE
LICENSE: Restore original Apache license

### PR DESCRIPTION
**Pull request by autoih**
* Remove copyright notice from first line. Apache's instruction: 

> Each original source document (code and documentation, but **excluding the LICENSE** and NOTICE files) SHOULD include a short license header at the top. 

Ref: http://www.apache.org/dev/apply-license.html#new

* The Apache 2.0 license includes an Appendix about how to apply the license terms to files. We should not modify the license file. 
Reference: https://github.com/kubernetes/kubernetes/commit/d30db1f9a915aa95402e1190461469a1889d92be

### CODE REVIEW

**Comment by ewilderj** [comment]
Thanks, I'm going to check this PR with our open source counsel.

**Comment by ewilderj** [comment]
This is the same situation as encountered in https://github.com/tensorflow/tensorflow/pull/26554 -- this change doesn't make any difference, so in the name of caution about changing the long-standing license doc, I'm closing this PR.

**Review by martinwicke — COMMENTED** [review]
You are correct about the appendix. 

But can you revert line 1? We checked with our lawyers, and that is actually correct.

**Review by martinwicke — CHANGES_REQUESTED** [review]
Please revert line 1

File: LICENSE [hunk 1]
```diff
@@ -1,4 +1,3 @@
-Copyright 2019 The TensorFlow Authors.  All rights reserved.
```
[the remark this answers is not in the document]
Comment (I-Hong) [reply LICENSE]: Thanks, @martinwicke. Updated!

**Comment by I-Hong** [comment]
Thanks, @martinwicke. Updated!

**Review by martinwicke — APPROVED** [review]
Thank you!

**Pull request merged**